hadoop-ozone/dist/src/main/compose/ozone-om-ha/docker-compose.yaml

```diff
@@ -36,7 +36,6 @@ services:
          - 9890:9872
       environment:
          ENSURE_OM_INITIALIZED: /data/metadata/om/current/VERSION
-         WAITFOR: scm:9876
```

Comment:
We removed WAITFOR env usage, there are few other files where this is used, om-statefulset.yaml. Do we need to remove from there also?

And also we are removing usage of WAITFOR, then do we need to remove the logic for this in docker image code?

Reply:
Making the changes to remove the wait for, @elek it makes sense to remove the wait for from everywhere, right?

Reply:
Yes, it should be fine AFAIK.